src/GetDirectory.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs');
+const path = require('path');
+const testFolder = path.dirname(require.main.filename)
+
+
+fs.readdirSync(testFolder).forEach(file => {
+  console.log(file);
+});
```

Comment:
Do we have any use for this? :confused:

Reply:
NO :P , just for testing purpose i was created this one